src/components/Chart/getOutlierSeries.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { splitOutlierSeries } from './getOutlierSeries'
+import { AxisDomain, ChartSeries } from './model'
+
+describe('splitOutlierSeries', () => {
+  const yDomain: AxisDomain = { init: 0, end: 100, step: 20 }
+  const xDomain: AxisDomain = { init: 0, end: 5, step: 1 }
+  const domainPoints: number[] = [0, 1, 2, 3, 4, 5]
+  const chartSeries: ChartSeries<number>[] = [{ name: 'test', data: [20, 50, 120] }]
+
+  it.each(['auto', 'expand-domain'] as const)(
+    'should get outlier series from series with outlier when request outliers based on the suportOutlier param',
+    (outliers: 'auto' | 'expand-domain') => {
```

Comment:
I think would be better if we define a type for this values

Reply:
Done